stories/Header/StickyHeader.js

```diff
@@ -0,0 +1,208 @@
+          </HeaderBrand>
+          <HeaderRightZone>
+            <HeaderSocialsZone label="Seguici su">
+              <ul>
+                <li>
+                  <a href="#" aria-label="Facebook" target="_blank">
+                    <Icon icon="it-facebook" />
+                  </a>
+                </li>
+                <li>
+                  <a href="#" aria-label="Github" target="_blank">
+                    <Icon icon="it-github" />
+                  </a>
+                </li>
+                <li>
+                  <a href="#" aria-label="Twitter" target="_blank">
+                    <Icon icon="it-twitter" />
+                  </a>
+                </li>
+              </ul>
+            </HeaderSocialsZone>
+            <HeaderSearch label="Cerca" iconName="it-search" />
+          </HeaderRightZone>
+        </HeaderContent>
+      </Header>
+    )
+  }
+}
+
+export class NavHeader extends Component {
+  state = {
+    collapsed: true
+  }
+
+  render() {
+    const isOpen = !this.state.collapsed
+    return (
+      // eslint-disable-next-line react/prop-types
+      <Header type="navbar" theme={this.props.theme}>
+        <HeaderContent expand="lg" megamenu>
+          <HeaderToggler
+            onClick={() => this.setState({ collapsed: isOpen })}
+            aria-controls="nav1"
+            aria-expanded="false"
+            aria-label="Toggle navigation">
+            <Icon icon="it-burger" />
+          </HeaderToggler>
+          <Collapse
+            isOpen={isOpen}
+            navbar
+            header
+            onOverlayClick={() => this.setState({ collapsed: isOpen })}>
+            <div className="menu-wrapper">
+              <Nav navbar>
+                <NavItem active>
+                  <NavLink href="#" active>
+                    <span>link 1 active </span>
+                    <span className="sr-only">current</span>
+                  </NavLink>
+                </NavItem>
+                <NavItem>
+                  <NavLink href="#" disabled>
+                    Link 2
+                  </NavLink>
+                </NavItem>
+                <NavItem>
+                  <NavLink href="#">Link 3</NavLink>
+                </NavItem>
+                <NavItem>
+                  <NavLink href="#">Link 4</NavLink>
+                </NavItem>
+                <NavItem>
+                  <NavLink href="#" disabled>
+                    Megamenu label
+                  </NavLink>
+                </NavItem>
+              </Nav>
+            </div>
+          </Collapse>
+        </HeaderContent>
+      </Header>
+    )
+  }
+}
+
+const StickyHeader = () => {
+  const open = boolean('show', true)
+  if (!open) {
+    return (
+      <Headers sticky>
+        <SlimHeader />
+        <CenterHeader />
+        <NavHeader />
+      </Headers>
+    )
+  } else {
+    return (
+      <Headers>
+        <SlimHeader />
+        <CenterHeader />
+        <NavHeader />
+      </Headers>
+    )
+  }
```

Comment:
A boolean prop can be expressed both with just the prop name and with the prop + boolean value.

This means you can rewrite this branching in:
```jsx
return (<Headers sticky={sticky}>
  <SlimHeader />
  <CenterHeader />
  <NavHeader />
</Headers>);
```